### DISCUSSION TITLE
Gophish landing page not working in windows

### DISCUSSION THREAD

**Discussion by CyberRangeUser** [gophish/gophish#3120] [Q&A]
![image](https://github.com/gophish/gophish/assets/165326999/319375f4-7cbf-46d5-9785-493d95f35a70)
Good evening. I can send messages but when i try to show a landing page by clicking in the link of a message, it does not work. The idea is to make a fake for educational purposes of gmail message. There i ask for the password. 
![image](https://github.com/gophish/gophish/assets/165326999/dc0b1540-ee9a-403e-a674-faa77aca7342)
This is the landing page:
![image](https://github.com/gophish/gophish/assets/165326999/3f9f4327-7456-4a7e-87db-7d6c077a27e3)
I have tried to to put in the campaign ip http:/0.0.0.0:80 because this is the phising server ip shown in the console
http://127.0.0.1:3333, ... Nothing worked. I am using windows so i installed it windows10. 
pd: I saw also that even if i open the emails the admin it doesn't show or apply this info.
Any help?

**Comment by g4xyk00** [comment]
Hi, you may try accessing the application at http://127.0.0.1 (HTTP instead of HTTPS at URL). 
You can check the IP address configured in the listen_url variable of the config.json file.
